tests/installation/encrypted_volume_activation.pm

```diff
@@ -0,0 +1,27 @@
+use base "basetest";
+use strict;
+use testapi;
+
+sub run {
+    assert_screen 'encrypted_volume_activation_prompt';
+    if (get_var('ENCRYPT_CANCEL_EXISTING')) {
+	send_key 'alt-c', 5;
```

Comment:
I miss interpreted do_wait as a timeout value, although I did look at the source code, so I have no excuse. The reason it is there in the first place though is because I was worried about getting a false positive. I am not sure if you are saying that will definitely not happen because openQA is too slow and will not move on to check_screen fast enough?